app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -96,19 +140,89 @@ export default {
     ID_PROVIDERS() {
       return IdentityProviders;
     },
+    designerOptions() {
```

Comment:
Going to move this to its own static file but there are multiple of us working on it so not yet.

Reply:
Yeah will discuss with Mina when she has a look at Basic mode. 👍